webapp/app/common/basic-schemas.ts

```diff
@@ -9,3 +9,11 @@ export const hexSchema = hexChars.transform((str) => str as Hex);
 export const addressSchema = hexChars
   .refine((str) => str.length === 42, "Address have to be 20 bytes long")
   .transform((str) => getAddress(str));
+
+export const numericStrSchema = z.coerce
+  .bigint({ message: "Must be a integer number" })
+  .transform((num) => num.toString());
```

Comment:
I'd rename this to `bigintStrSchema` to not confuse it with any future `numericStrSchema` that's actually of number type.

Reply:
Renamed!